notebooks/214-vision-paddle-classification/214-vision-paddle-classification.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5206ff2e-8ee6-47b0-abef-eb1b7f9ec5ee",
+   "metadata": {},
+   "source": [
+    "# PaddlePaddle Image Classification with OpenVINO\n",
+    "This demo shows how to run a MobileNetV3 Large PaddePaddle model using OpenVINO Runtime. Instead of exporting the PaddlePaddle model to ONNX and converting to Intermediate Representation (IR) format using Model Optimizer, we can now read the Paddle model directly without conversion."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d3dbb77-5cd6-46ad-b6e4-ba75f0ee795f",
+   "metadata": {},
+   "source": [
+    "## Download the MobileNetV3_large_x1_0 Model\n",
+    "Download the pre-trained model directly from the server. More details about the pre-trained model can be found in the PaddleClas documentation below.\n",
+    "\n",
+    "Source: https://github.com/PaddlePaddle/PaddleClas/blob/release/2.2/deploy/lite/readme_en.md"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d1d476b3-0d6e-420b-9e51-9a1629bda494",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "import os\n",
+    "import urllib.request\n",
+    "import tarfile\n",
+    "\n",
+    "mobilenet_url = \"https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/inference/MobileNetV3_large_x1_0_infer.tar\"\n",
+    "mobilenetv3_model_path = Path(\"model/MobileNetV3_large_x1_0_infer/inference.pdmodel\")\n",
+    "if mobilenetv3_model_path.is_file(): \n",
+    "    print(\"Model MobileNetV3_large_x1_0 already exists\")\n",
+    "else:\n",
+    "    # Download the model from the server, and untar it.\n",
+    "    print(\"Downloading the MobileNetV3_large_x1_0_infer model (20Mb)... May take a while...\")\n",
+    "    # create a directory \n",
+    "    os.makedirs('model')\n",
```

Comment:
please define string variable using `"` instead of `'`

Reply:
@OpenVINO-dev-contest please use ". For consistency of the notebooks.